storage/remote/kafka/kafka.proto

```diff
@@ -0,0 +1,12 @@
+package kafka;
+
+message Sample {
+	message Tag {
+		required string name = 1;
+		required string value = 2;
+	}
+
+	required double value = 1;
+	required int64 timestamp = 2;
+	repeated Tag tags = 3;
+}
```

Comment:
So basically we already have a canonical protobuf representation of samples in the Prometheus universe:
https://github.com/prometheus/client_model/blob/master/metrics.proto

I don't know if that properly works with Kafka too, but if so, it would probably be better not to define another one on top.